src/app/core/header/header.component.spec.ts

```diff
@@ -5,8 +5,42 @@ import { HeaderModule } from './header.module';
 
 describe('HeaderComponent', () => {
   let shallow: Shallow<HeaderComponent>;
+  const filterClass = '.filter-labels';
 
   beforeEach(() => {
     shallow = new Shallow(HeaderComponent, HeaderModule);
   });
+
+  it('should emit downloadClicked when download button is clicked', async () => {
+    const { find, outputs } = await shallow.render();
+    find('.download').triggerEventHandler('click', {});
+    expect(outputs.downloadClicked.emit).toHaveBeenCalled();
+  });
+
+  it('should emit logoClicked when logo is clicked', async () => {
+    const { find, outputs } = await shallow.render();
+    find('.logo').triggerEventHandler('click', {});
+    expect(outputs.logoClicked.emit).toHaveBeenCalled();
+  });
+
+  it('should display the current sex', async () => {
```

Comment:
This and the other two tests are very similar. Should be refactored into a utility test function (see below).
```ts
function testFilterLabel(index, data, result) {
  find = shallow.render({ bind: { filter: data } })
  label = find(`.filter-labels:nth-child(${index})`).nativeElement
  expect(label).toBe(result)
}

it('...', {
  testFilterLabel(1, { sex: 'Both' }, 'Sex: Both')
})
```